pom.xml

```diff
@@ -7,7 +7,7 @@
     <version>${product.version}</version>
     <name>Virtual Schema JDBC Adapter</name>
     <properties>
-        <product.version>4.0.2</product.version>
+        <product.version>4.0.1</product.version>
```

Comment:
We need to get rid of the variable here soon, since having a variable in the version is deprecated.

Reply:
removed